lib/game.rb

```diff
@@ -97,4 +97,18 @@ def self.average_goals_by_season
     end
     goals_per_season
   end
-end
\ No newline at end of file
+
+  def self.highest_total_score
```

Comment:
Great class methods for calculating the highest total score and lowest total score!